tests/unit/Utopia/Response/Filters/V16Test.php

```diff
@@ -0,0 +1,700 @@
+                    'filesCreate' => [],
+                    'filesRead' => [],
+                    'filesUpdate' => [],
+                    'filesDelete' => [],
+                ],
+            ],
+            'storage usage' => [
+                [
+                    'filesStorage' => $metrics,
+                ],
+                [
+                    'storage' => $metrics,
+                    'bucketsCreate' => [],
+                    'bucketsRead' => [],
+                    'bucketsUpdate' => [],
+                    'bucketsDelete' => [],
+                    'filesCreate' => [],
+                    'filesRead' => [],
+                    'filesUpdate' => [],
+                    'filesDelete' => [],
+                ],
+            ],
+        ];
+    }
+
+    /**
+     * @dataProvider usageStorageProvider
+     */
+    public function testUsageStorage(array $content, array $expected): void
+    {
+        $model = Response::MODEL_USAGE_STORAGE;
+
+        $result = $this->filter->parse($content, $model);
+
+        $this->assertEquals($expected, $result);
+    }
+
+    public function usageUsersProvider(): array
+    {
+        $metrics = [
+            [
+                'value' => 123,
+                'date' => Model::TYPE_DATETIME_EXAMPLE,
+            ]
+        ];
+        return [
+            'usersCount usage' => [
+                [
+                    'usersTotal' => $metrics,
+                ],
+                [
+                    'usersCount' => $metrics,
+                    'usersCreate' => [],
+                    'usersRead' => [],
+                    'usersUpdate' => [],
+                    'usersDelete' => [],
+                    'sessionsProviderCreate' => [],
+                    'sessionsDelete' => [],
+                ],
+            ],
+            'sessionsCreate usage' => [
+                [
+                    'sessionsTotal' => $metrics,
+                ],
+                [
+                    'sessionsCreate' => $metrics,
+                    'usersCreate' => [],
+                    'usersRead' => [],
+                    'usersUpdate' => [],
+                    'usersDelete' => [],
+                    'sessionsProviderCreate' => [],
+                    'sessionsDelete' => [],
+                ],
+            ],
+        ];
+    }
+
+    /**
+     * @dataProvider usageUsersProvider
+     */
+    public function testUsageUsers(array $content, array $expected): void
+    {
+        $model = Response::MODEL_USAGE_USERS;
+
+        $result = $this->filter->parse($content, $model);
+
+        $this->assertEquals($expected, $result);
+    }
+
+    public function usageVariableProvider(): array
+    {
+        return [
+            'functionId' => [
+                [
+                    'resourceId' => '5e5ea5c16897e',
+                ],
+                [
+                    'functionId' => '5e5ea5c16897e',
+                ],
+            ],
+        ];
+    }
+
```

Comment:
Let's remove it if we're not using it now, we can always get it back from git if needed

```suggestion
```